apps/dashboard/src/components/subscribers/subscriber-list.tsx

```diff
@@ -181,7 +178,7 @@ export const SubscriberList = (props: SubscriberListProps) => {
         toggleSort={toggleSort}
       >
         {data.data.map((subscriber) => (
-          <SubscriberRow key={subscriber.subscriberId} subscriber={subscriber} />
+          <SubscriberRow key={subscriber.subscriberId} subscriber={subscriber} subscribersCount={data.data.length} />
```

Comment:
Please rebase and make sure to have the `key={subscriber.id}` instead of `subscriber.subscriberId`